incubator/kubeless/templates/ui-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-ui
+  namespace: kubeless
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+    controller: {{ template "fullname" . }}-ui
```

Comment:
Is this label required? Suffix is `ui`. Normally I would expect a label `component: ui` then.

Reply:
nop, it seems it isn't. I changed to `component: ui`